tests/simulator/initialize/test_fields_init_2d.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+from ddt import ddt, data, unpack
+from pyphare.core.box import Box, Box2D, nDBox
+from tests.simulator.test_initialization import InitializationTest
+
+import matplotlib
+
+matplotlib.use("Agg")  # for systems without GUI
+
+ndim = 2
```

Comment:
I think we should add a comment explaining why this file is needed as it could be seen as duplication from the _1d test.

Reply:
commented top of all per dim files